js/components/ImportIndicatorsPopover.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { BootstrapPopoverButton } from './helpPopover';
+import api from '../apiv2';
+
+class ImportIndicatorsPopover extends React.Component {
+    ONE = 0;
+    TWO = 1;
+    constructor(props) {
+        super(props);
+        this.state = {
+            status: this.ONE
+        }
+    }
+
+    handleDownload = () => {
+        console.log("Download Clicked");
+        api.downloadTemplate()
+            .then(response => {
+                console.log("Reponse:", response);
+                alert("DONWLOAD TEMPLATE")
+            })
+    }
+    handleUpload = () => {
+        console.log("Upload Clicked");
+        api.uploadTemplate()
+        .then(response => {
+            console.log("Reponse:", response);
+            alert("UPLOAD TEMPLATE")
+        })
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                {(() => {
+                    switch(this.state.status) {
+                        case this.ONE:
+                            return (
+                                <div className="importIndicators-body">
+                                    <div className="import-text">
+                                        <ol>
+                                            <li>
+                                                {
+                                                    // # Translators: TODO
```

Comment:
Did you mean to do these translator comments?

Reply:
I added these translator comments. Also, I had merge hatchling a few commits back but just merged hatchling again today just in case. I didn't get any conflicts.